### ISSUE TITLE
A11y_AzureDataStudio_CreateAzureKubernetesServiceCluster_Keyboard: Add text & Add code buttons are not accessible by the keyboard.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7623]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:** #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG2.1.1;#WCAG2.4.7;#DesktopApp;#Win10;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Repro Steps:**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on 
        select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next 
        button and click on script to Notebook.
8.	 " Create Azure Kubernetes service cluster" screen is opened and navigate to on the screen using tab.
 9. Check the "Add code and Add text" buttons are accessible by the keyboard or not.

**Actual:**
Add code & Add text buttons are not accessible by the keyboard.

**Expected:**
Keyboard focus should move on "Add text & Add code" buttons and these buttons should accessible by the keyboard.

**Same issue observed for the below scenario:**

**1. Prerequisites** 
1. Install /Launch the application Azure data studio.
2. Navigate to connections and click on more action button.
3. Select “Deploy SQL Server big data cluster” from more action drop down.
4. Click on “Deploy SQL Server big data cluster”.
5. Check the check box I accept in the “select the deployment option” screen and click on select button.
6. Select any block and click on next button and again click on next button.
7. Enter password and confirm password and click on next button and again click on next button and click on script to Notebook ->Navigate till Prerequisites.
8. Verify that all the elements of " Prerequisites" are accessible and MAS Compliant.

**2. Check dependencies**
1. Install /Launch the application Azure data studio.
2. Navigate to connections and click on more action button.
3. Select “Deploy SQL Server big data cluster” from more action drop down.
4. Click on “Deploy SQL Server big data cluster”.
5. Check the check box I accept in the “select the deployment option” screen and click on select button.
6. Select any block and click on next button and again click on next button.
7. Enter password and confirm password and click on next button and again click on next button and click on script to Notebook ->Navigate till Check dependencies.
8. Verify that all the elements of " Check dependencies " are accessible and MAS Compliant

**User Impact:**
Keyboard only user faced difficulty to access the control.

**Attachment for Reference:**
[7623_A11y_AzureDataStudio_CreateAzureKubernetesServiceCluster_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3711970/7623_A11y_AzureDataStudio_CreateAzureKubernetesServiceCluster_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
@corivera  @chlafreniere could you please take a look?

**Comment by corivera** [comment]
@chlafreniere I'm guessing making the buttons always visible would fix this. Does that sound alright to you?

**Comment by alanrenmsft** [comment]
@corivera I think we should be good as long as it is available in the menus. I'll leave it to you guys to follow up with the testing team.

**Comment by chlafreniere** [comment]
It's in the menus and we also have key combos for creating new code and new text cells.

**Comment by alanrenmsft** [comment]
@vinodbk197 closing since the feature is in the menu '...' menu of the cell.

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-May2019;
